releng/org.eclipse.cdt.lsp.target/org.eclipse.cdt.lsp.target.target

```diff
@@ -81,6 +81,18 @@
 					<version>1.23.0</version>
 					<type>jar</type>
 				</dependency>
+				<dependency>
+					<groupId>org.apache.commons</groupId>
+					<artifactId>commons-text</artifactId>
+					<version>1.10.0</version>
+					<type>jar</type>
+				</dependency>
+				<dependency>
+					<groupId>org.apache.commons</groupId>
+					<artifactId>commons-lang3</artifactId>
```

Comment:
Is `commons-lang3` needed by `commons-text`?

Reply:
Yes, not adding this dependency results in a runtime exception.